src/qobj/arithmetic_and_attributes.jl

```diff
@@ -538,17 +558,20 @@ function _ptrace_ket(QO::AbstractArray, dims::Union{SVector,MVector}, sel)
     qtrace = filter(i -> i ∉ sel, 1:nd)
     dkeep = dims[sel]
     dtrace = dims[qtrace]
-    # Concatenate sel and qtrace without loosing the length information
-    sel_qtrace = ntuple(Val(nd)) do i
-        if i <= length(sel)
-            @inbounds sel[i]
+    nt = length(dtrace)
+
+    # Concatenate qtrace and sel without loosing the length information
```

Comment:
Can you fix my typo `loosing` with `losing`?

Reply:
done